pkg/core/mempool/mem_pool.go

```diff
@@ -69,8 +70,13 @@ type Pool struct {
 
 	resendThreshold uint32
 	resendFunc      func(*transaction.Transaction, interface{})
-	// removeStaleCallback is a callback method which is called after item is removed from the mempool.
-	removeStaleCallback func(*transaction.Transaction, interface{})
+
+	// subscriptions for mempool events
+	subscriptionsEnabled atomic.Bool
```

Comment:
It is provided on construction, I don't think there is a need for atomic here.

Reply:
Fixed.